src/Data/Config.php

```diff
@@ -165,24 +165,39 @@ static function ( $query ) {
 	 * @return string
 	 */
 	public function graphql_wp_query_cursor_pagination_stability( string $orderby, WP_Query $wp_query ) {
-
+		// Bail early if it's not a GraphQL Request.
 		if ( true !== is_graphql_request() ) {
 			return $orderby;
 		}
 
-		global $wpdb;
+		// If pre-filter hooked, return $pre_orderby.
+		$pre_orderby = apply_filters( 'graphql_pre_wp_query_cursor_pagination_stability', null, $orderby, $wp_query );
+		if ( null !== $pre_orderby ) {
+			return $pre_orderby;
+		}
 
-		// If the cursor "graphql_cursor_compare" arg is not in the query,
-		// default to using ID DESC as the stabilizer
-		if ( ! isset( $wp_query->query['graphql_cursor_compare'] ) ) {
-			return "{$orderby}, {$wpdb->posts}.ID DESC ";
+		// Bail early if disabled by connection.
+		if ( isset( $wp_query->query_vars['graphql_apply_cursor_pagination_orderby'] )
```

Comment:
Is the change from `$query->query` to `$query->query_vars` intentional.

(This seems like a valid change to me, just wondering about any downstream breaks).